modules/ROOT/pages/managing-connected-apps.adoc-

```diff
@@ -0,0 +1,19 @@
+= Managing Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Connected apps are applications that allow external applications to securely interact with the Anypoint Platform APIs by providing controlled access to its resources using OAuth 2.0 authentication protocols, without the need to share user credentials directly. For more information, see xref:access-management::connected-apps-overview.adoc[].
+
+When an asset is created using a connected app and is published to Exchange, the asset owner is automatically set as the owner of the connected app. 
```

Comment:
`the asset owner is automatically set as the owner of the connected app. `
I think this might be ambiguos for a first time reader.  I think it can be misinterpreted to mean that the asset owner now owns the connected app.
Can this be rephrased to avoid that misinterpretation?